### ISSUE TITLE
Seems as though you have been TOS_BLOCKed, unable to refresh QR Code.

### ISSUE THREAD

**Issue by TVARecordings** [open-wa/wa-automate-nodejs#3123]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Personal account (normal)

### Mode

EASY API/CLI

### Current Behavior

I've updated to the latest version, and now it throw error: "Seems as though you have been TOS_BLOCKed, unable to refresh QR Code.". It says to visit https://github.com/open-wa/wa-automate-nodejs#best-practice, which leads to a dead end.

### Expected Behavior

No errors :-)

### Steps To Reproduce

npm start

### create() code OR full CLI command + CONFIG

```markdown
0 verbose cli C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js
1 info using npm@9.6.4
2 info using node@v20.0.0
3 timing npm:load:whichnode Completed in 1ms
4 timing config:load:defaults Completed in 1ms
5 timing config:load:file:C:\Program Files\nodejs\node_modules\npm\npmrc Completed in 1ms
6 timing config:load:builtin Completed in 1ms
7 timing config:load:cli Completed in 2ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:D:\rzkytmgr Bot-SadBotBeforeAfterUpdate\.npmrc Completed in 1ms
10 timing config:load:project Completed in 1ms
11 timing config:load:file:C:\Users\TomVa\.npmrc Completed in 0ms
12 timing config:load:user Completed in 0ms
13 timing config:load:file:C:\Users\TomVa\AppData\Roaming\npm\etc\npmrc Completed in 0ms
14 timing config:load:global Completed in 0ms
15 timing config:load:setEnvs Completed in 1ms
16 timing config:load Completed in 6ms
17 timing npm:load:configload Completed in 6ms
18 timing npm:load:mkdirpcache Completed in 0ms
19 timing npm:load:mkdirplogs Completed in 0ms
20 verbose title npm create LgAuAC4A or npx @open-wa/wa-automate xml text
21 verbose argv "create" "-encodedCommand" "LgAuAC4A" "or" "npx" "@open-wa/wa-automate" "--debug" "--loglevel" "verbose" "-inputFormat" "xml" "-outputFormat" "text"
22 timing npm:load:setTitle Completed in 1ms
23 timing config:load:flatten Completed in 2ms
24 timing npm:load:display Completed in 5ms
25 verbose logfile logs-max:10 dir:C:\Users\TomVa\AppData\Local\npm-cache\_logs\2023-05-16T23_34_42_068Z-
26 verbose logfile C:\Users\TomVa\AppData\Local\npm-cache\_logs\2023-05-16T23_34_42_068Z-debug-0.log
27 timing npm:load:logFile Completed in 6ms
28 timing npm:load:timers Completed in 0ms
29 timing npm:load:configScope Completed in 0ms
30 timing npm:load Completed in 19ms
31 timing arborist:ctor Completed in 1ms
32 silly logfile start cleaning logs, removing 2 files
33 silly logfile done cleaning log files
34 timing arborist:ctor Completed in 0ms
35 http fetch GET 404 https://registry.npmjs.org/create-LgAuAC4A 746ms (cache skip)
36 http fetch GET 404 https://registry.npmjs.org/create-LgAuAC4A 700ms (cache skip)
37 timing command:create Completed in 1731ms
38 verbose stack HttpErrorGeneral: 404 Not Found - GET https://registry.npmjs.org/create-LgAuAC4A - Not found
38 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\npm-registry-fetch\lib\check-response.js:95:15
38 verbose stack     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
38 verbose stack     at async RegistryFetcher.packument (C:\Program Files\nodejs\node_modules\npm\node_modules\pacote\lib\registry.js:87:19)
38 verbose stack     at async RegistryFetcher.manifest (C:\Program Files\nodejs\node_modules\npm\node_modules\pacote\lib\registry.js:118:23)
38 verbose stack     at async getManifest (C:\Program Files\nodejs\node_modules\npm\node_modules\libnpmexec\lib\index.js:30:22)
38 verbose stack     at async missingFromTree (C:\Program Files\nodejs\node_modules\npm\node_modules\libnpmexec\lib\index.js:56:22)
38 verbose stack     at async C:\Program Files\nodejs\node_modules\npm\node_modules\libnpmexec\lib\index.js:165:32
38 verbose stack     at async Promise.all (index 0)
38 verbose stack     at async exec (C:\Program Files\nodejs\node_modules\npm\node_modules\libnpmexec\lib\index.js:163:3)
38 verbose stack     at async Init.execCreate (C:\Program Files\nodejs\node_modules\npm\lib\commands\init.js:134:5)
39 verbose statusCode 404
40 verbose pkgid create-LgAuAC4A@*
41 verbose cwd D:\rzkytmgr Bot-SadBotBeforeAfterUpdate
42 verbose Windows_NT 10.0.19045
43 verbose node v20.0.0
44 verbose npm  v9.6.4
45 error code E404
46 error 404 Not Found - GET https://registry.npmjs.org/create-LgAuAC4A - Not found
47 error 404
48 error 404  'create-LgAuAC4A@*' is not in this registry.
49 error 404 This package name is not valid, because 
50 error 404  1. name can no longer contain capital letters
51 error 404
51 error 404 Note that you can also install from a
52 error 404 tarball, folder, http url, or git url.
53 verbose exit 1
54 timing npm Completed in 1786ms
55 verbose code 1
56 error A complete log of this run can be found in: C:\Users\TomVa\AppData\Local\npm-cache\_logs\2023-05-16T23_34_42_068Z-debug-0.log
```

### DEBUG INFO

```markdown
PS D:\rzkytmgr Bot-SadBotBeforeAfterUpdate> npm start

> whatsapp-bot@1.0.0 start
> node index.js

2023-05-17 1:29:23
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃                                                                                                   ┃
┃   @open-wa/wa-automate                                                                            ┃
┃    💬 🤖 The most reliable NodeJS whatsapp library for chatbots with advanced features.           ┃
┃   Version: 2.1.6                                                                                  ┃
┃   Check out the latest changes: https://github.com/open-wa/wa-automate-nodejs#latest-changes      ┃
┃                                                                                                   ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
- Initializing WA

  Puppeteer old Headless deprecation warning:
    In the near feature `headless: true` will default to the new Headless mode
    for Chrome instead of the old Headless implementation. For more
    information, please see https://developer.chrome.com/articles/new-headless/.
    Consider opting in early by passing `headless: "new"` to `puppeteer.launch()`
    If you encounter any bugs, please report them to https://github.com/puppeteer/puppeteer/issues/new/choose.

- Browser Launched
┌─────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│       (index)       │                                                                    Values                                                                    │
├─────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│     WA_VERSION      │                                                     'I think you have been TOS_BLOCKed'                                                      │
│       PAGE_UA       │ 'WhatsApp/2.2037.6 Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36' │
│ WA_AUTOMATE_VERSION │                                                       '2.1.6 UPDATE AVAILABLE: 4.66.0'                                                       │
│   BROWSER_VERSION   │                                                        'HeadlessChrome/112.0.5614.0'                                                         │
└─────────────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
- Authenticating
- Authenticate to continue
- Authenticate to continue
- Loading QR
- Authenticate to continue
- Loading QR
```

### Environment

```markdown
- OS: Windows 10
- Node: Latest version
- npm: latest version
```

**Comment by nrs1022** [comment]
It seems your version is really old (2.1.6), it says:

    │ WA_AUTOMATE_VERSION │                                                       '2.1.6 UPDATE AVAILABLE: 4.66.0'                                                       │

**Comment by TVARecordings** [comment]
> It seems your version is really old (2.1.6), it says:
> 
> ```
> │ WA_AUTOMATE_VERSION │                                                       '2.1.6 UPDATE AVAILABLE: 4.66.0'                                                       │
> ```
> 
> You should update to latest version.

Thanks, i thought i had update it. I guess i didn't do it properly
